src/tooltip.js

```diff
@@ -217,6 +218,18 @@ angular.module('ts.utils')
           }
         }
         else{
+          angular.element($window).bind('resize', function(){
```

Comment:
angularElement.bind seems to be deprecated (https://docs.angularjs.org/api/ng/function/angular.element) they suggest to use .on instead. 

I'm afraid this will leak the listener when the element unmounts.